src/BuildIt.Forms/BuildIt.Forms.Controls/BuildIt.Forms.Controls.csproj

```diff
@@ -82,4 +82,17 @@
       <Version>2.0.0</Version>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="StatefulControl.xaml">
```

Comment:
Just checking - are these items required? I thought they're automatically included in the new project format.

Reply:
You're probably right. VS is adding these automatically. I will remove them